lsbfdump-jni-rs/native/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "lsbfdump"
+version = "0.1.0"
+authors = ["Shane Dell <shanedell100@gmail.com>"]
+edition = "2021"
```

Comment:
2021 ??

Reply:
Yeah so I just made this using `cargo new <project-name>`. That is what it auto sets  `edition` to, if you try changing it, it will most likely cause things to break. The only valid values I could find for `edition` was 2015, 2018 and 2021.

Reply:
Note that `edition` isn't supposed to be the year this application was made, which I think Mike was implying. It's basically the version of the rust language to use. It's like saying to use C89, C99, C11, etc. for the C language. Same core language, but different capabilities/keywords/etc.

https://doc.rust-lang.org/edition-guide/editions/